atm.js

```diff
@@ -1,21 +1,44 @@
-var checkingBalance = document.querySelector("div#checking_balance");
-checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
 
-var checkingDeposit = function(amount){
-  checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
-
-}
-
-function checkingWithdrawal(amount){
-
-}
-
-function savingsDeposit(amount){
-
-}
-
-function savingsWithdrawal(amount){
-
-}
-
-// an eventListerner for each button, each one a "click"
\ No newline at end of file
+$(function(){
+
+	var cbalance = parseFloat( $("#checking .balance").text().replace("$", "") );
+	var sbalance = parseFloat( $("#saving .balance").text().replace("$", "") );
+
+	$( "#checking .depositbtn" ).on( "click", function(){
+		var cInput = parseFloat( $("#checking .balanceinput").val().replace("$", "") );
+		cbalance = cInput + cbalance;
+		$("#checking .balance").text('$' + cbalance);
+	});
+
+	$( "#checking .withdrawbtn" ).on( "click", function(){
+		var cInput = parseFloat( $("#checking .balanceinput").val().replace("$", "") );
+		cbalance = cbalance - cInput;
+		if(cbalance < 0){
+			if(sbalance > 0){
+				sbalance = sbalance - Math.abs(cbalance);
+				if(sbalance < 0){
+					cbalance = sbalance;
+					sbalance = 0;
+				}else{
+					cbalance = 0;
+				}
+
+				$("#saving .balance").text('$' + sbalance);
+			}
+		}
+		$("#checking .balance").text('$' + cbalance);
```

Comment:
It would be nice if the user were notified (with an `alert()`) if they didn't have enough money to perform a transaction. Currently this will fail silently.